Sources/_OpenAPIGeneratorCore/Translator/Responses/TypedResponseHeader.swift

```diff
@@ -164,3 +164,8 @@ extension FileTranslator {
         )
     }
 }
+
+/// An unresolved OpenAPI response header.
+///
+/// Can be either a reference or an inline response header.
+typealias UnresolvedHeader = Either<JSONReference<OpenAPI.Header>, OpenAPI.Header>
```

Comment:
Nit: The OpenAPI spec has the headers at a scope not specific to requests, e.g. when used in `#components/headers`.

```suggestion
/// An unresolved OpenAPI header.
///
/// Can be either a reference or an inline header.
typealias UnresolvedHeader = Either<JSONReference<OpenAPI.Header>, OpenAPI.Header>
```

Reply:
Hmm I think @andrewse02 had it correctly, `Header Object` ([OpenAPI specification link](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#header-object)) _must not_ have `in` and `name` fields, which makes it invalid to provide in a place of a parameter. So my understanding has always been that they only apply to response headers, and request headers are just parameters. Are you reading the specification differently?

Reply:
Yep. I checked the spec before commenting. I was considering parity with how it’s spelled out in the spec and with the type name chosen here. 

Seemed reasonable to keep them aligned.

Reply:
I see – we kind of have two options, because we use the explicit name "response header" all over the generator codebase, to explicitly disambiguate it from request headers, which are just parameters. So the question is whether we want to stick with the spec spelling, or with the spelling we use in the generator business logic. Either is fine, I guess, but we should probably rename things holistically across the codebase to avoid drift.

Reply:
Ah, do we. I hadn't remembered that and didn't notice it from the context of the diff. Let's just leave this as it is then.

Reply:
Oh @czechboy0—were you potentially suggesting we actually rename the type alias here to match the rest of the generator? i.e. this:
```suggestion
/// An unresolved OpenAPI response header.
///
/// Can be either a reference or an inline response header.
typealias UnresolvedResponseHeader = Either<JSONReference<OpenAPI.Header>, OpenAPI.Header>
```

Reply:
Might be worth to go all the way and disambiguate everywhere, but I'd also be happy with that being a separate issue, to avoid further expanding this PR. It'll likely be more invasive than the scope of this PR.

Reply:
Broke it out: https://github.com/apple/swift-openapi-generator/issues/78